#!/usr/bin/env python
PACKAGE = "path_track"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#gen.add("int_param",int_t,0,"int parameter",1,0,10)
#gen.add("inflation_radius",double_t,0,"double parameter",0.6,0.0,2.0)
gen.add("pid_p",double_t,0,"double parameter",1,0,5)
gen.add("pid_i",double_t,0,"double parameter",0,0,5)
gen.add("pid_d",double_t,0,"double parameter",0,0,5)
gen.add("ave_vel",double_t,0,"double parameter",0.3,0,3)


exit(gen.generate(PACKAGE,"point_cloud_test","path_track_"))
